updatem/orchestration/update_orchestrator_feedback.go

```diff
@@ -116,32 +131,28 @@ func handleDomainIdentificationFailed(orchestrator *updateOrchestrator, domain,
 	orchestrator.updateStatusIdentificationFailed(domain, message)
 }
 
-func handleDomainCompletedEvent(orchestrator *updateOrchestrator, domain, message string) {
-	if orchestrator.operation.status != types.StatusRunning {
-		return
-	}
+func handleDomainCompletedEvent(orchestrator *updateOrchestrator, domain, message string, actions []*types.Action) {
 	domainStatus := orchestrator.operation.domains[domain]
-	if domainStatus == types.StatusCompleted || domainStatus == types.StatusIncomplete {
+	if domainStatus == types.StatusCompleted || domainStatus == types.StatusIncomplete ||
+		domainStatus == types.BaselineStatusCleanupSuccess || domainStatus == types.BaselineStatusCleanupFailure {
 		return
 	}
-	orchestrator.operation.domains[domain] = types.StatusCompleted
-	orchestrator.command(context.Background(), orchestrator.operation.activityID, domain, types.CommandCleanup)
+	orchestrator.operation.domains[domain] = types.BaselineStatusCleanupSuccess
```

Comment:
The function is called _handleDomainCompletedEvent_. Is _types.BaselineStatusCleanupSuccess_ the correct domain status?

Reply:
It is a little misleading, but actually this is the current internal domain status here. When the Update Manager receives a COMPLETED event for a specific domain, it must not send any CLEANUP commands to this domain. COMPLETED event is considered a terminal status. That's the reason why in this case the domain status is marked as baselineStatusCleanupSuccess.